doc/source/getting_started/troubleshooting.rst

```diff
@@ -32,6 +32,12 @@ Your credential to access huggingface can be found online at `https://huggingfac
 
 You can set the token as an environmental variable, with ``export HUGGING_FACE_HUB_TOKEN=your_token_here``.
 
+Download models from ModelScope
+-------------------------------
```

Comment:
`Download models from ModelScope` should not be a section under `No huggingface repo access`. To fix it, use `=` instead of `-`.